port/zos390/omrosdump.c

```diff
@@ -333,6 +338,9 @@ tdump(struct OMRPortLibrary *portLibrary, char *asciiLabel, char *ebcdicLabel, u
 		char dsn[256];
 	} *dsnPattern31;
 
+	/* Defined in omrgenerate_ieat_dump.s */
```

Comment:
Please finish each sentence with a period.

Reply:
Added missing period, also rebased to latest again, please review.